rules/windows/lateral_movement_execution_via_file_shares_sequence.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/03"
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of a file that was created by the virtual System process. This may indicate lateral movement
```

```suggestion
Identifies the execution of a file that was created by the virtual system process. This may indicate lateral movement
```
or `SYSTEM` if related to privs